content/reference/admin/private_locations/configuration/azure/index.md

```diff
@@ -68,6 +72,12 @@ control-plane {
       # Use "subnets.name" as returned by Azure CLI:
       # az network vnet subnet list --resource-group MyResourceGroup --vnet-name MyVNet
       subnet-name = "default"
+      
+      # Overwrite JAVA_HOME definition (optional)
+      # java-home = "/usr/lib/jvm/zulu17"
+      
+      # JVM Options (optional)
+      # jvm-options = ["-Xmx6G"]
```

Comment:
Shouldn't we explain:
- what the defaults are
- which options we recommend keeping from the defaults (most of them)
- what user will most likely want to change (Xms, Xmx)
?

Reply:
edit: I guess this is applied on top of default options, rather than replacing them entirely - but that should be explained (instead of my second bullet point).